tests/test_processors.py

```diff
@@ -23,8 +32,68 @@ def _get_processor_file(self, processor):
                             'processors',
                             processor.strip() + '.py')
 
+    @staticmethod
+    def _get_first_data(processor, parameters, data_in,
+                        dp_out, data_out, env):
+        '''Returns the first line of data_out as a python object.'''
+        process = subprocess.run([sys.executable, processor, '1',
```

Comment:
I was thinking something along these lines.
Right now there's an equality check for the output data - if you could override just that, perhaps you could implement a more complicated comparison logic without having to rewrite the entire class.